frontend/src/metabase/search/components/SearchSidebar/SearchSidebar.tsx

```diff
@@ -22,11 +21,37 @@ export const SearchSidebar = ({
   value: SearchFilters;
   onChangeFilters: (filters: SearchFilters) => void;
 }) => {
+  const filterMap: Record<FilterTypeKeys, SearchFilterComponent | null> =
+    useMemo(
+      () => ({
+        [SearchFilterKeys.Type]: TypeFilter,
+        [SearchFilterKeys.Verified]: PLUGIN_CONTENT_VERIFICATION.VerifiedFilter,
+      }),
+      [],
+    );
+
+  const isValidFilterValue = useCallback(
```

Comment:
Might move this to a `validate` function on a per-filter basis in the next iteration. Let me know what you think, maybe it should be done now.